src/main/java/io/anserini/fusion/FuseRuns.java

```diff
@@ -0,0 +1,237 @@
+/*
+ * Anserini: A Lucene toolkit for reproducible information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.anserini.fusion;
+
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+import org.kohsuke.args4j.ParserProperties;
+import org.kohsuke.args4j.spi.StringArrayOptionHandler;
+
+class FusedRunOutputWriter implements Closeable {
+  private final PrintWriter out;
+  private final String format;
+  private final String runtag;
+
+  public FusedRunOutputWriter(String output, String format, String runtag) throws IOException {
+    this.out = new PrintWriter(Files.newBufferedWriter(Paths.get(output), StandardCharsets.UTF_8));
+    this.format = format;
+    this.runtag = runtag;
+  }
+
+  private class Document implements Comparable<Document>{
+    String docid;
+    double score;
+
+    public Document(String docid, double score)
+    {
+      this.docid = docid;
+      this.score = score;
+    }
+
+    @Override public int compareTo(Document a)
+    {
+      return Double.compare(a.score,this.score);
+    }
+
+  }
+
+  public void writeTopic(String qid, HashMap<String,Double> results) {
```

Comment:
```suggestion
  public void writeTopic(String qid, Map<String,Double> results) {
```

No need to specify impl